report/4.Design/HardwareTest.tex

```diff
@@ -0,0 +1,24 @@
+\section{Hardware testing}
+To ensure the most accurate results, it is important to ensure the quality and precision of the chosen hardware to be at its highest.
+Therefore, a hardware test has been performed on the motors, checking their precision and quality.
```

Comment:
Think this should be "Therefore, a test of the motors has been conducted, ...." or something